.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - opened
+      - synchronize
+jobs:
+  build:
+    strategy:
+      fail-fast: false
```

Comment:
This will force all the builds to run, regardless of whether one of them fails. That isn't the default GH Action behaviour.

Reply:
👍